doc/documentation.tex

```diff
@@ -466,6 +417,105 @@ \subsubsection{Methods comparison}
+All algorithms had tendency to match areas where the point density was the highest. An example of this behavior may be observed using \textit{Config21.json}. It uses two "airbus" clouds with decreased sizes and added noise and outliers. Result achieved by algorithms are presented below and vary among each other.
+
+\begin{figure}[H]
+	\centering
+    \subfloat[\centering CPD]{{\includegraphics[width=5cm]{mr-cpd-21.png} }}
+    \hspace{0.5em}
+    \subfloat[\centering ICP]{{\includegraphics[width=5cm]{mr-icp-21.png} }}
+    \hspace{0.5em}
+    \subfloat[\centering NICP]{{\includegraphics[width=5cm]{mr-nicp-21.png} }}
+    \caption{Tests with \textit{Config21.json}}
+\end{figure}
+
+The ICP did not manage to converge in this example but result achieved by CPD is interesting. Algorithm connected wheels and engines of both airplanes. Feasibly that is because those are areas with highest point density in entire cloud. NICP returned exact transformation.
+
+Similar situation was met using \textit{Config11.json} and \textit{Config18.json}. Unfortunately in tests using the latter ICP returned identity matrix. The results are presented below.
+
+\begin{figure}[H]
+	\centering
+    \subfloat[\centering CPD]{{\includegraphics[width=5cm]{mr-cpd-11.png} }}
+    \hspace{0.5em}
+    \subfloat[\centering ICP]{{\includegraphics[width=5cm]{mr-icp-11.png} }}
+    \hspace{0.5em}
+    \subfloat[\centering NICP]{{\includegraphics[width=5cm]{mr-nicp-11.png} }}
+    \caption{Tests with \textit{Config11.json}}
+\end{figure}
+
+\begin{figure}[H]
+	\centering
+    \subfloat[\centering CPD]{{\includegraphics[width=6cm]{mr-cpd-18.png} }}
+    \hspace{0.5em}
+    \subfloat[\centering NICP]{{\includegraphics[width=6cm]{mr-nicp-18.png} }}
+    \caption{Tests with \textit{Config18.json}}
+\end{figure}
+
+In contrary to \textit{Config18.json} and \textit{Config21.json}, using \textit{Config11.json} CPD almost found exact solution while this time NICP focused on matching areas with highest density.
+
+Another example of CPD accuracy in tough conditions can be found when using \textit{Config26.json}. We used two "airbus" clouds where first had no wheels, engines and vertical stabilizer. The second one had wings and wheels cut. Also we have added noise and outliers to both clouds. Results are presented below.
+
+\begin{figure}[H]
+	\centering
+    \subfloat[\centering CPD]{{\includegraphics[width=5cm]{mr-cpd-26.png} }}
+    \hspace{0.5em}
+    \subfloat[\centering ICP]{{\includegraphics[width=5cm]{mr-icp-26.png} }}
+    \hspace{0.5em}
+    \subfloat[\centering NICP]{{\includegraphics[width=5cm]{mr-nicp-26.png} }}
+    \caption{Tests with \textit{Config26.json}}
+\end{figure}
+
+CPD found exact solution. As mentioned above ICP has trouble with incomplete clouds and it remains unchanged with this configuration. NICP was close to finding exact solution; however, it lacks of $90\degree$ rotation.
```

Comment:
- the exact solution
- ',' after mentioned above
- trouble -> troubles

Reply:
Done